portfolio/src/main/java/com/google/sps/servlets/RandomArtist.java

```diff
@@ -25,8 +25,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
               add("The Weeknd");
               add("Anderson .Paak");
               add("Hozier");
-              add("Daniel Caeasr");
-              add("Jacon Collier");
+              add("Daniel Caesar");
```

Comment:
Looks like some of your typo corrections for #11 made it into this pull request as well - no worries, you can just submit this one and delete the other one if you need to